build: avoid using bundled shapelib
[asterios.dramis@gmail.com: imported the patch from Fedora into openSUSE's librecad]

---
 plugins/importshp/importshp.pro |   11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

Index: LibreCAD-2.0.8/plugins/importshp/importshp.pro
===================================================================
--- LibreCAD-2.0.8.orig/plugins/importshp/importshp.pro
+++ LibreCAD-2.0.8/plugins/importshp/importshp.pro
@@ -17,12 +17,9 @@ include(../../common.pri)
 # For plugins
 INCLUDEPATH    += ../../librecad/src/plugins
 
-SOURCES += importshp.cpp \
-           shapelib/shpopen.c \
-           shapelib/safileio.c \
-           shapelib/dbfopen.c
-HEADERS += importshp.h \
-           shapelib/shapefil.h
+SOURCES += importshp.cpp
+HEADERS += importshp.h
+LIBS += -lshp
 
 win32 {
         DLLDESTDIR = ../../windows/resources/plugins
@@ -36,5 +33,3 @@ unix {
 	TARGET = ../../unix/resources/plugins/$$PLUGIN_NAME
     }
 }
-
-INCLUDEPATH    += shapelib
